src/java/org/apache/cassandra/db/compaction/OperationType.java

```diff
@@ -70,13 +89,20 @@ public static OperationType fromFileName(String fileName)
         throw new IllegalArgumentException("Invalid fileName for operation type: " + fileName);
     }
 
+    public boolean isCacheSave()
+    {
+        return this == COUNTER_CACHE_SAVE || this == KEY_CACHE_SAVE || this == ROW_CACHE_SAVE;
+    }
+
     public String toString()
     {
         return type;
     }
 
+    public static final Predicate<OperationType> EXCEPT_VALIDATIONS = o -> o != VALIDATION;
+    public static final Predicate<OperationType> COMPACTIONS_ONLY = o -> o == COMPACTION || o == TOMBSTONE_COMPACTION;
     public static final Predicate<OperationType> REWRITES_SSTABLES = o -> o == COMPACTION || o == CLEANUP || o == SCRUB ||
                                                                           o == TOMBSTONE_COMPACTION || o == ANTICOMPACTION ||
                                                                           o == UPGRADE_SSTABLES || o == RELOCATE ||
                                                                           o == GARBAGE_COLLECT;
-}
+}
```

Comment:
Nit: I suggest to restore this new line at the end of file (unnecessary change)

Reply:
Actually, I think we should merge https://github.com/datastax/cassandra/pull/295
and then use ds-trunk version when rebasing

Reply:
This one wasn't resolved. Or do I miss to see the update in GH?

Reply:
my local diff doesn't show anything on this file.. not sure why Github shows this

Reply:
works now